API_README.md

```diff
@@ -1,8 +1,19 @@
-![img](./public/images/openemr-rest-api.png)
+# OpenEMR REST API Documentation
 
-### Overview
+## Overview
 
-Easy-to-use JSON-based REST API for OpenEMR. All code is done in classes and separate from the view to help with codebase modernization efforts. FHIR is also supported.
+Easy-to-use JSON-based REST API for OpenEMR. All code is done in classes and separate from the view to help with codebase modernization efforts. FHIR is also supported, see FHIR API documentation [here](FHIR_README.md)
+
+### Sections
+* [facility API](https://github.com/openemr/openemr/blob/master/API_README.md#post-apifacility)
+* [provider API](https://github.com/openemr/openemr/blob/master/API_README.md#get-apiprovider)
+* [patient API](https://github.com/openemr/openemr/blob/master/API_README.md#post-apipatient)
+* [insurance API](https://github.com/openemr/openemr/blob/master/API_README.md#get-apipatientpidinsurance)
+* [appointment API](https://github.com/openemr/openemr/blob/master/API_README.md#get-apiappointment)
+* [document API](https://github.com/openemr/openemr/blob/master/API_README.md#get-apipatientpiddocument)
+* [message API](https://github.com/openemr/openemr/blob/master/API_README.md#post-apipatientpidmessage)
+* [dev notes](https://github.com/openemr/openemr/blob/master/API_README.md#dev-notes)
+* [todos](https://github.com/openemr/openemr/blob/master/API_README.md#project-management)
```

Comment:
Having anchors is a good idea, but need to do them without requiring full path like this (or things will break when have this in different release branches). Is there a way to do this without requiring the full paths?

Reply:
Yes there is @bradymiller . I just need to add the relative file paths. Thanks.

Reply:
Hello @bradymiller, fixed